app/components/UI/Stake/components/StakingEarnings/StakingEarningsHistory/StakingEarningsHistoryList/StakingEarningsHistoryList.tsx

```diff
@@ -0,0 +1,111 @@
+import React from 'react';
+import { View } from 'react-native';
+import Label from '../../../../../../../component-library/components/Form/Label';
+import Text from '../../../../../../../component-library/components/Texts/Text';
+import { TextVariant } from '../../../../../../../component-library/components/Texts/Text/Text.types';
+import { useTheme } from '../../../../../../../util/theme';
+import styleSheet from './StakingEarningsHistoryList.styles';
+import SkeletonPlaceholder from 'react-native-skeleton-placeholder';
+
+interface StakingEarningsHistoryListProps {
+  earnings: StakingEarningsHistoryListData[];
+  filterByGroupLabel?: string;
+}
+
+export interface StakingEarningsHistoryListData {
+  label: string;
+  amount: string;
+  amountSecondaryText: string;
+  groupLabel: string;
+  groupHeader: string;
+  ticker: string;
+}
+
+const StakingEarningsHistoryList = ({
+  earnings,
+  filterByGroupLabel,
+}: StakingEarningsHistoryListProps) => {
+  const { colors } = useTheme();
+  const styles = styleSheet();
+  let lastGroupHeader: string | null = null;
+  return (
+    <View style={styles.stakingEarningsHistoryListContainer}>
+      {earnings ? (
+        <>
+          <Label variant={TextVariant.BodyMDBold} color={colors.text.default}>
+            Payout history
```

Comment:
Might be best to replace hardcoded strings with translation variables. Apologies for not catching this on the last review!

```suggestion
            {strings("stake.payout_history")}
```